src/flow-worker.c

```diff
@@ -195,9 +195,10 @@ TmEcode FlowWorker(ThreadVars *tv, Packet *p, void *data, PacketQueue *preq, Pac
                 Detect(tv, x, detect_thread, NULL, NULL);
                 FLOWWORKER_PROFILING_END(x, PROFILE_FLOWWORKER_DETECT);
             }
-#if 0
+
             //  Outputs
-#endif
+            OutputLoggerLog(tv, x, fw->output_thread, preq, unused);
```

Comment:
it seems a bit odd to pass on 'unused' here, as that makes it 'used' :)

If we need it, then we should rename it. Otherwise, lets replace it's use here by NULL or remove the argument from the OutputLoggerLog func completely.